### ISSUE TITLE
SimICE: changed register pairs

### ISSUE THREAD

**Issue by stefandrissen** [simonowen/simcoupe#55]
Is this an intentional "optimisation" or just a side effect of showing changes in registers only?

![image](https://user-images.githubusercontent.com/5297464/125688059-3b8d10aa-4a9f-467a-b09f-f098e050b97b.png)

While I see the benefit of showing less noise, the downside is that I need to trace back to the previous change of ``h`` to find which address is actually being read or written.

**Comment by simonowen** [comment]
I _think_ it's always done it like that, and is just showing the changed register parts. Currently it can't tell the difference between `INC L` and `INC HL`. It would be good if the disassembler could give a breakdown of the instruction to report what it's using, to help guide what is shown for differences.

If your version is old enough you can use Space to toggle the display of the full double register. I did take that out recently when making other trace changes, but I should probably restore it until there's a better alternative.

**Comment by stefandrissen** [comment]
I was not aware of space, so that will do the job nicely! I did see the change in the trace display a few days ago, so that probably subliminally triggered this issue.  😀

**Comment by simonowen** [comment]
The CPU core change introduced a trace bug where it thought more of the register had changed, so the double register was always shown. At the same time that was "fixed" I limited the Space behaviour to only affect AF, and documented the key shortcut.

I'll restore the original behaviour until there's something better to replace it. I've got a few buffered changes so I'll create an updated release soon. Though it'll probably remain a dev branch release until I've restored some of the lost Raspberry Pi performance.
